_books/ion-1-1/src/whats_new.md

```diff
@@ -210,6 +69,7 @@ a (:values 1 2 3) b (:none) c
 a 1 2 3 b c
 ```
 
+**E-expressions in lists or S-expressions**
 <!-- TODO: Move this (or copy it) to a dedicated section on e-expression expansion -->
```

Comment:
Should this TODO be deleted?

Reply:
Is this now said dedicated section on e-expression expansion? I suppose I don't know where this line came from.